DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,198 @@
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get(".modal-content").should("be.visible");
+
+    })
+
+    it('cannot submit with all empty', () => {
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#firstName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("label[for='gender-radio-1']").should('have.css','border-color','rgb(220, 53, 69)')
+        cy.get("#userNumber").should('have.css','border-color','rgb(220, 53, 69)')
+    })
+
+    it('cannot submit without FirstName', () => {
+
+        cy.get("#lastName").type("Vo")
+        cy.get("label[for='gender-radio-1']").click();
+        cy.get("#userNumber").type("0933008846")
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#firstName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+
+    })
+
+    it('cannot submit without LastName empty', () => {
+
+        cy.get("#firstName").type("Phuong")
+        cy.get("label[for='gender-radio-1']").click();
+        cy.get("#userNumber").type("0933008846")
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+
+    })
+
+    it('cannot submit with firstName and LastName have specChar ', () => {
+
+        cy.get("#firstName").type("Phuong$")
+        cy.get("#lastName").type("Vo*")
+        cy.get("label[for='gender-radio-2']").click();
+        cy.get("#userNumber").type("0933008846")
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#first").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+
+    })
+
+
+    it('cannot submit without chosing gender', () => {
+        cy.get("#firstName").type("Phuong")
+        cy.get("#lastName").type("Vo")
+        cy.get("#userNumber").type("0933008846")
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("label[for='gender-radio-1']").should('have.css','border-color','rgb(220, 53, 69)')
+
+    })
+
+    it('cannot submit without mobile', () => {
+        cy.get("#firstName").type("Phuong")
+        cy.get("#lastName").type("Vo")
+        cy.get("label[for='gender-radio-1']").click();
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#userNumber").should('have.css','border-color','rgb(220, 53, 69)');
+
+    })
+
+    it('cannot submit without mobile less than 10 digits', () => {
+        cy.get("#firstName").type("Phuong")
+        cy.get("#lastName").type("Vo")
+        cy.get("label[for='gender-radio-1']").click();
+        cy.get("#userNumber").type("093300884")
+
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#userNumber").should('have.css','border-color','rgb(220, 53, 69)');
+
+    })
+
+    it('cannot submit with mobile inculed only char', () => {
```

Comment:
this case and `cannot submit with mobile include char and digits` can combine into `cannot submit when mobile format is not correct`, and use fixture file to create a list and loop through test data inside